src/test/resources/application.conf

```diff
@@ -22,23 +22,42 @@ createCaseUrl = "caseworkers/"${userCcdId}"/jurisdictions/PROBATE/case-types/Gra
 getUserProfileUrl = "caseworkers/538/profile"
 
 localhost {
-  maxSimulationDurationMinutes = 99999
-  userCcdId = 538
+  maxSimulationDurationMinutes = 1
+  maxResponseTime = 20000
+  meanResponseTime = 1000
+  minthinktime = 0
+  maxthinktime = 0
+  minWaitForNextIteration = 1
+  maxWaitForNextIteration = 2
+  totalDuration = 1
+  reachRPSTarget = 20
+  reachRPSDuration = 10
+  reachRPSHoldForDuration = 60
+  jumptoRPSTarget = 50
+  jumptoRPSDuration = 900
+  userCcdId = 131068
   userImportId = 1
   userProfileUrl = "http://localhost:4453"
   caseDataUrl = "http://localhost:4452"
   caseDefinitionUrl = "http://localhost:4451"
   idamS2SUrl = "http://localhost:4502"
   idamUserTokenUrl = "http://localhost:4501"
-  dataStoreS2STokenGeneratorSecret = "${IDAM_KEY_CCD_DATA_STORE}"
-  gatewayS2STokenGeneratorSecret = "${IDAM_KEY_CCD_GATEWAY}"
+  dataStoreS2STokenGeneratorSecret = "AAAAAAAAAAAAAAAB"
```

Comment:
can you put back the Env variables here please?

Reply:
will do